metadata/service/aggregator/single.yml

```diff
@@ -6,3 +6,4 @@ parameters:
   heka:
     aggregator:
       enabled: true
+      timezone: ${_param:linux_timezone}
```

Comment:
BTW I wasn't really happy with the use of `salt['pillar.get']('linux:system:timezone')` across all formulas as we had in the previous proposal but it's probably ok if it's only happening in this file. What do you think?